Tests/RevenueCatUITests/Templates/OnePackageWithFeaturesPaywallViewTests.swift

```diff
@@ -0,0 +1,22 @@
+import Nimble
+import RevenueCat
+@testable import RevenueCatUI
+import SnapshotTesting
+
+#if !os(macOS)
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *)
+class OnePackageWithFeaturesPaywallViewTests: BaseSnapshotTest {
+
+    func testSamplePaywall() {
+        let view = PaywallView(offering: Self.offering.withLocalImages,
+                               introEligibility: Self.eligibleChecker,
+                               purchaseHandler: Self.purchaseHandler)
+        view.snapshot(size: Self.fullScreenSize)
+    }
+
+    private static let offering = TestData.offeringWithSinglePackageFeaturesPaywall
+
+}
+
+#endif
```

Comment:
the screenshot looks like it's going over safe areas?

Reply:
Because there's no safe area in the screenshot :) But if you test on a device you see it looks good.